src/match/tasks.py

```diff
@@ -1,75 +1,60 @@
-import datetime
-import logging
-import os
-import sys
-
-from celery import Celery
-from celery.signals import after_setup_logger
-from django.utils import timezone
-from django_celery_beat.models import IntervalSchedule
-from django_celery_beat.models import PeriodicTask
-from match import match_status
-from match.controllers.create_match_controller import CreateMatchController
-from match.models import Match
-from person.models import Person
-
-# Get Pear algorithm
-current_dir = os.path.dirname(os.path.abspath(__file__))
-submodule_path = f"{current_dir}/../../pear-algorithm/src"
-sys.path.insert(0, submodule_path)
-from main import main
-
-app = Celery()
-
-# https://www.distributedpython.com/2018/08/28/celery-logging/
-logger = logging.getLogger(__name__)
-
-
-@after_setup_logger.connect
-def setup_loggers(logger, *args, **kwargs):
-    formatter = logging.Formatter(
-        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
-    )
-    sh = logging.StreamHandler()
-    sh.setFormatter(formatter)
-    logger.addHandler(sh)
-
-
-@app.task
-def matcher():
-    # Cancel previous matches that were not completed. TODO verify this is what we want to do
-    today = timezone.now()
-    last_week = today - datetime.timedelta(
-        days=7
-    )  # should be before pears are released but after matching
-    previous_matches = Match.objects.filter(created_date__range=[last_week, today])
-    print(f"Previous: {previous_matches}")
-    unfinished_matches = previous_matches.exclude(
-        status__in=[match_status.CANCELED, match_status.INACTIVE]
-    )
-    print(f"Unfinished: {unfinished_matches}")
-    unfinished_matches.update(status=match_status.CANCELED)
-
-    # Generate pears
-    users = Person.objects.all()
-    pears = main(users, logger)
-    match_creator = CreateMatchController(None, None, return_status=True)
-
-    # Create new matches
-    for pear in pears:
-        match_creator._data = {"ids": pear}
-        success, error_msg = match_creator.process()
-        if not success:
-            print(f"Match error between {pear}: {error_msg}")
-
-
-schedule, _ = IntervalSchedule.objects.get_or_create(
-    every=1,
-    period=IntervalSchedule.MINUTES,
-)
-
-PeriodicTask.objects.get_or_create(
-    interval=schedule,
-    name="Pear Algorithm",
-    task="match.tasks.matcher",
-)
+# # Get Pear algorithm
+# current_dir = os.path.dirname(os.path.abspath(__file__))
+# submodule_path = f"{current_dir}/../../pear-algorithm/src"
+# sys.path.insert(0, submodule_path)
+# from main import main
+
+# app = Celery()
+
```

Comment:
why is this file all commented out?

Reply:
this is my fault, this is gonzalo's previous implementation of matching which I improved on a different branch & PR [(noah/auto-matcher)](https://github.com/cuappdev/pear-django-backend/pull/97). at my meeting tomorrow with @chalo2000 and @jzs27 we're going to merge in that branch, deploy to dev, and see if it works

tl;dr: this file was messing up our other celery stuff so I should've just deleted it alltogether

Reply:
sg